src/test/kotlin/test/tests/hashers/CodeLongevityTest.kt

```diff
@@ -353,4 +351,123 @@ class CodeLongevityTest : Spek({
             testRepo.destroy()
         }
     }
+
+    given("'line storage #1'") {
```

Comment:
This and next test group fails with java.io.FileNotFoundException if data/storage directory not presented.